packages/cashier/src/stores/general-store.ts

```diff
@@ -258,19 +258,13 @@ export default class GeneralStore extends BaseStore {
         }
     }
 
-    async getAdvertizerError() {
-        const advertiser_info = await this.WS.authorized.p2pAdvertiserInfo?.();
-        this.setP2pAdvertiserError(getPropertyValue(advertiser_info, ['error', 'code']));
-    }
+    async checkP2pStatus(): Promise<void> {
+        const get_account_status: Promise<AccountStatusResponse> = this.WS.wait(
+            'get_account_status'
+        ) as unknown as Promise<AccountStatusResponse>;
```

Comment:
```suggestion
        const get_account_status = this.WS.wait('get_account_status');
```

@ameerul-deriv Update the `wait` type in `packages/cashier/src/types/websocket.types.ts` as following:
```ts
wait: <T extends TSocketEndpointNames>(value: T) => Promise<TSocketResponse<T>>;
```